manifest.toml

```
@@ -0,0 +1,71 @@
+packaging_format = 2
+
+id = "roundcube"
+name = "Roundcube"
+description.en = "Open Source Webmail software"
+description.fr = "Webmail Open Source"
+
+version = "1.6.5~ynh1"
+
+maintainers = []
+
+[upstream]
+license = "GPL-3.0-only"
+website = "https://roundcube.net/"
+demo = "https://demo.yunohost.org/webmail/"
+admindoc = "https://github.com/roundcube/roundcubemail/wiki"
+code = "https://github.com/roundcube/roundcubemail"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/webmail"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.language]
+    ask.en = "Choose the application language"
+    ask.fr = "Choisissez la langue de l'application"
+    type = "select"
+    choices = ["de_DE", "en_GB", "fr_FR", "it_IT"]
+    default = "en_GB"
+
+    [install.with_carddav]
+    ask.en = "Install CardDAV synchronization plugin?"
+    ask.fr = "Installer le plugin de synchronisation CardDAV ?"
+    type = "boolean"
+    default = false
+
+[resources]
+        [resources.sources.main]
+        url = "https://github.com/roundcube/roundcubemail/releases/download/1.6.5/roundcubemail-1.6.5.tar.gz"
+        sha256 = "6c830faa0674d917c426d9b694793743c51f8bf649cd6e7fc605957bcccd1730"
+        autoupdate.strategy = "latest_github_tag"
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "mariadb-server, php-pear, php8.2-ldap, php8.1-mysql, php8.1-cli, php8.1-intl, php8.1-zip, php8.1-gd, php8.1-mbstring, php8.1-dom, php8.1-curl"
```

```suggestion
    packages = "mariadb-server, php-pear, php8.1-ldap, php8.1-mysql, php8.1-cli, php8.1-intl, php8.1-zip, php8.1-gd, php8.1-mbstring, php8.1-dom, php8.1-curl"
```

8.2 -> 8.1 for consistency